### ISSUE TITLE
Web interface for scraper data

### ISSUE THREAD

**Issue by bowswung** [bowswung/voobly-scraper#1]
At present the data this scraper collects is only available in CSV format. 

A web interface to this data would allow easy access to this data to any interested parties.

The minimal feature set would be to provide a way of viewing match data, with aggregated stats, and custom filtering based on the available data (civs, elo etc).

For example, it should be possible to construct a query like:
"Show all 1v1 Arabia matches between Teutons and Franks in the last 3 months played with UP1.6 between players ranked 1600-2000"

The site should be structured in such a way as to support the addition of two major features in the future:

1) Analysis of game recordings (possibly using https://github.com/goto-bus-stop/recanalyst) with data from this analysis used for categorising games and therefore for filtering. E.g. "All games with scout rushes".

2) A mirror/archive of recorded games from Voobly.

**Comment by bowswung** [comment]
http://aocrecs.com - this is doing something similar.

**Comment by atomfried** [comment]
Hi!

> A web interface to this data would allow easy access to this data to any interested parties.
> 
> The minimal feature set would be to provide a way of viewing match data, with aggregated stats, and custom filtering based on the available data (civs, elo etc).

Anyone working on that already? I'm thinking about building a platform for match statistics and I found your project...

**Comment by bowswung** [comment]
Yes there are efforts afoot! There is a Discord focused on this and other related projects: https://discord.gg/q56y3PT and a GitHub organisation: https://github.com/SiegeEngineers
